### PR TITLE
PMM-5058 CONTRIBUTING.md copied to Master from PMM-2

### PR DESCRIPTION
**Pull request by dbazhenov**
I copied CONTRIBUTING.md from PMM-2.0 branch to master because master is the main branch and it is seen by all new users.
Jira issue:
https://jira.percona.com/browse/PMM-5058

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,3 +16,42 @@ The easiest way to make a local development setup is to use Docker Compose.
...
+
+## Workflow for the contributor
+
+1.  Find the task in [JIRA](https://jira.percona.com/issues/?jql=project+%3D+PMM+AND+component+%3D+%22Grafana+Dashboards%22) or create a new task. Use a component filter Grafana Dashboards.
+
+2.  You need to make a fork of our repository in your GitHub account.
+
+3.  Make a clone of your repository on your computer.
+
+4.  Switch the repository to the correct branch. Now it's PMM-2.0.
```
Comment (atymchuk) [inline CONTRIBUTING.md:28]: Our master branch points to PMM-1.x

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,3 +16,42 @@ The easiest way to make a local development setup is to use Docker Compose.
...
+        git add .
+        git commit -m "PMM-5053 Tooltip for Head Block widget for Prometheus Dashboard was added"
+
+8.  Push your branch into your repository. Check that your branch has only the necessary code changes. 
+
+        git push origin PMM-5053_dbazhenov_tooltip
+
+9.  Make a Pull Request from your branch to the right branch in percona/grafana-dashboards.
+
+    Example: from dbazhenov:PMM-5053_dbazhenov_tooltip to percona:PMM-2.0
```
Comment (atymchuk) [inline CONTRIBUTING.md:47]: PMM-1.x

**Comment by rnovikovP** [comment]
@dbazhenov   I would suggest to  NOT  encourage people to submit  PR  to PMM1 branch. 

w can do here as we did for PMM   repo 
image
 and route everyone to the PMM2  branch!

@sanjabytes , PTAL

**Comment by rnovikovP** [comment]
ok. made  https://github.com/percona/grafana-dashboards/pull/678  to explain what I think we should do  with this and other repos

**Pull request closed without merging**